src/javascript/app/pages/user/new_account/digital_options.js

```diff
@@ -0,0 +1,77 @@
+const BinarySocket     = require('../../../base/socket');
+const BinaryPjax       = require('../../../base/binary_pjax');
+const Client           = require('../../../base/client');
+const getElementById   = require('../../../../_common/common_functions').getElementById;
+const ClientBase       = require('../../../../_common/base/client_base');
+const urlFor           = require('../../../../_common/url').urlFor;
+const showLoadingImage = require('../../../../_common/utility').showLoadingImage;
+
+const DigitalOptions = (() => {
+
+    let is_virtual,
+        el_welcome_container,
+        upgrade_info,
+        is_uk,
+        is_unwelcome_uk,
+        synthetics,
+        financial;
+
+    const init = () => {
+        is_virtual            = Client.get('is_virtual');
+        upgrade_info          = ClientBase.getBasicUpgradeInfo();
+        el_welcome_container  = getElementById('welcome_container');
+        is_uk                 = Client.get('residence') === 'gb';
+        is_unwelcome_uk       = Client.isUnwelcomeUk();
+        synthetics            = getElementById('default');
+        financial             = getElementById('financial');
```

Comment:
We use `$` prefix as a reminder that this is a jQuery object throughout the code. 

For vanilla javascript objects, you can use `el_` prefix.